mmv1/products/cgc/terraform.yaml

```diff
@@ -51,6 +51,26 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           deletion_protection: "false"
         ignore_read_extra:
           - "deletion_protection"
+      - !ruby/object:Provider::Terraform::Examples
```

Comment:
Apologies for the delay, lost track of a few PR's.
The tests failed again, these examples all need a couple of more lines:
```
        ignore_read_extra:
          - "deletion_protection"
```

Reply:
done